microprofile-health/pom.xml

```diff
@@ -13,7 +13,7 @@
   </parent>
 
   <artifactId>microprofile-health</artifactId>
-  <packaging>jar</packaging>
+  <packaging>ejb</packaging>
```

Comment:
with this documentation change shouldn't the pom packaging stay as jar?

Reply:
ok, think I understand now, the generated pom.xml has no such change, right?

Reply:
There is no way to generate the packaging directly as ejb, so users for step-by-step instructions need to do the change themselves.

This change is required as the QS itself is consumed in the JB studio. They don't follow step-by-step AFAIK.

Reply:
please change to war